build/serve.js

```diff
@@ -134,7 +134,7 @@ gulp.task('spawn-backend', ['backend', 'kill-backend'], function () {
  * backend process is killed beforehand, if any. In production the backend does serve the frontend
  * pages as well.
  */
-gulp.task('spawn-backend:prod', ['build-frontend', 'backend:prod', 'kill-backend'], function () {
+gulp.task('spawn-backend:prod', ['build-frontend', 'backend:prod', 'kill-backend', 'wait-for-cluster'], function () {
```

Comment:
I'm not sure that 'wait-for-cluster' belongs here. Spawn backend should just spawn the backend. Whoever uses this task and also requires local cluster should wait for it.

How about moving this to local-up-cluster? I.e., the task starts cluster and waits till it is ready. And then order the tasks in integration test